addons/proton_scatter/src/cache/scatter_cache.gd

```diff
@@ -74,8 +66,9 @@ func _ready() -> void:
 		# Set the cache path to the cache folder, incorporating the scene name
 		cache_file = DEFAULT_CACHE_FOLDER.get_basename().path_join(scene_name + "_scatter_cache.res")
 		return
-
-	restore_cache.call_deferred()
+	
+	
+	call_deferred("_try_restore_cache")
```

Comment:
Better use the new syntax `_try_restore_cache.call_deferred()` (to get proper code completion and avoid string operations)